docs/src/pages/api-reference/server.mdx

```diff
@@ -149,6 +149,80 @@ accessible here
 The UploadThing API Helper, for use ON YOUR SERVER. It's basically just a REST
 API but better
 
+### `uploadFiles`
+
+`uploadFiles` let's you upload files directly from your server, without having
+to define file routes. When uploadthing files from the server, no presigned URLs
+are used, your files are uploaded directly to S3 via our servers.
```

Comment:
> your files are uploaded directly to S3 via our servers

- "directly" is not really correct here
- S3 bit is unnecessary, the backing storage doesn't matter to the end user. 

I think this could be changed to:

"your files are uploaded via our servers"